internal/service/s3/object_data_source.go

```diff
@@ -45,6 +43,7 @@ func DataSourceObject() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			// TODO checksum_mode etc.
```

Comment:
Leftover TODO, or will this be in a separate PR?

Reply:
Coming soon in this PR...